Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -404,6 +404,21 @@ def add_comment_to_reverted_commit_bug_tracker(cls, repository, args, pr, commit
             issue.open(why='Reverted by {}'.format(pr.url))
         return 0
 
+    @classmethod
+    def add_comment_to_issue(cls, issue, update_issue, pr):
+        if issue and update_issue:
```

Comment:
I feel like having a function both take these arguments (especially update_issue, which I believe to be a bool) and then immediately have an `if` branch is kinda surprising.

Like, as I understand it, the interface is currently:

```python
def add_comment_to_issue(cls, issue: Tracker | None, update_issue: bool, pr: PullRequest) -> None: ...
```

Whereas I think I'd expect:

```python
def add_comment_to_issue(cls, issue: Tracker, pr: PullRequest) -> None: ...
```

And then it's up the caller to ensure it's not None, and to only call it when the update is wanted?

Reply:
Great point - I think this weird interface was due to me not thinking and just copy-pasting existing code into a function.